spec/support/components/work_packages/relations.rb

```diff
@@ -252,6 +252,15 @@ def expect_relation(relatable)
         find_row(relatable)
       end
 
+      def expect_closest_relation(relatable)
+        actual_relatable = find_relatable(relatable)
+        relation_row = find_row(actual_relatable)
+
+        within relation_row do
+          expect(page).to have_css(".Label", text: "Closest")
+        end
```

Comment:
With the new `it` of ruby 3.4, this can be rewritten like this:

```suggestion
        within relation_row do
          expect(it).to have_css(".Label", text: "Closest")
        end
```

Not that useful, but I find it neat in conjunction with `within`.

Reply:
the `it` implicit parameter is a feature that I make frequent use of in Kotlin/Android, so definitely nice to see it in Ruby.